src/app/routes/enrolment/view-requests/components/expiration-date/expiration-date.component.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+
+import { ExpirationDateComponent } from './expiration-date.component';
+import { DebugElement, NO_ERRORS_SCHEMA } from '@angular/core';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { ReactiveFormsModule } from '@angular/forms';
+import { MatInputModule } from '@angular/material/input';
+import { MatIconTestingModule } from '@angular/material/icon/testing';
+import { MatButtonModule } from '@angular/material/button';
+import { MatDatepickerModule } from '@angular/material/datepicker';
+import { MatNativeDateModule } from '@angular/material/core';
+import { dispatchInputEvent, getElement } from '@tests';
+
+describe('ExpirationDateComponent', () => {
+  let component: ExpirationDateComponent;
+  let fixture: ComponentFixture<ExpirationDateComponent>;
+  let hostDebug: DebugElement;
+  let baseTime: Date;
+  let timezoneSeconds: number;
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        declarations: [ExpirationDateComponent],
+        imports: [
+          ReactiveFormsModule,
+          MatInputModule,
+          MatIconTestingModule,
+          MatButtonModule,
+          NoopAnimationsModule,
+          MatDatepickerModule,
+          MatNativeDateModule,
+        ],
+        schemas: [NO_ERRORS_SCHEMA],
+      }).compileComponents();
+    })
+  );
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ExpirationDateComponent);
+    component = fixture.componentInstance;
+    hostDebug = fixture.debugElement;
+    spyOn(component.add, 'emit');
+    jasmine.clock().install();
+    baseTime = new Date(2000, 0, 1, 1, 0, 0);
+    timezoneSeconds = baseTime.getTimezoneOffset() * 60;
+    jasmine.clock().mockDate(baseTime);
+  });
+
+  afterEach(() => {
+    jasmine.clock().uninstall();
+  });
+
+  it('should create', () => {
+    fixture.detectChanges();
+    expect(component).toBeTruthy();
+  });
+
+  // TODO: passing locally, failing on GHA. Check it.
+  xit('should set expiration date and reset it', () => {
```

Comment:
Is this something you want to do in this PR or should we make a separate task to investigate and fix?

Reply:
Ok, sounds good. Could you add a new task to SB backlog so that we don't forget please?